### ISSUE TITLE
Fix four digit gas price

### ISSUE THREAD

**Issue by Baschdl** [WGierke/informatiCup2018#48]
Is this a comma fault or a very high prediction? 
```python
{'end': (48.401030000000006, 9.98764), 'name': ['Aral Tankstelle'], 'start': (52.390530000000005, 13.064540000000001), 'address': ['B 100\n6796 Brehna'], 'overall_price': 42005.76141429667, 'fill_liters': [32.512199237071727], 'prices': [1292.0], 'payment': [42005.76141429667], 'stops': [(51.5537376, 12.1966591)]}
```

**Comment by WGierke** [comment]
Possibly introduced [here](https://github.com/WGierke/informatiCup2018/commit/a34c380f1ae83fc0506cb04b855eac4582165301#diff-f4904c00e43cf6282c04ecc5541bf469L61)

**Comment by WGierke** [comment]
Fixed via #49
